cves/CVE-2018-13302.yml

```diff
@@ -144,10 +150,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: 'was not found via unit tests'
+  fix: false
+  fix_answer: 'commit had no changes to tests'
```

Comment:
I would also mention that modern code coverage is not checking the lines that were added
http://coverage.ffmpeg.org/index.src_libavformat_movenc.c.html